Unix/etc/omiserver.conf

```diff
@@ -16,51 +16,36 @@
 #idletimeout=TIMEOUT
 
 ##
-## trace -- enable tracing to standard output (default is 'false')
-##
-#trace=(true|false)
-
-##
-## loglevel -- set the log level of the server
+## loglevel -- set the loggiing options for MI server
+##   Valid options are: ERROR, WARNING, INFO, DEBUG, VERBOSE (debug builds only)
+##   If commented out, then default value is: WARNING
 ##
 loglevel = WARNING
 
 ##
-## <NICKNAME> -- set the value of nickname.
+## This section is for security protocol settings
+##   Including NoSSLv2,NoSSLv3,NoTLSv1_0,NoTLSv1_1,NoTLSv1_2 and NoSSLCompression.
 ##
-#prefix=PATH
-#libdir=PATH
-#bindir=PATH
-#localstatedir=PATH
-#sysconfdir=PATH
-#providerdir=PATH
-#certsdir=PATH
-#datadir=PATH
-#rundir=PATH
-#logdir=PATH
-#schemadir=PATH
-#schemafile=PATH
-#pidfile=PATH
-#logfile=PATH
-#registerdir=PATH
-#pemfile=PATH
-#keyfile=PATH
-#agentprogram=PATH
-#serverprogram=PATH
-#includedir=PATH
-#configfile=PATH
 NoSSLv2=true
 NoSSLv3=false
 NoTLSv1_0=false
```

Comment:
For lines 31-34, I think these have been dragged along forever. I don't think we give guidance to change these. If we don't, we may want to pull them out. Or, if you leave them, then at least include comments on what, EXACTLY, each one does.

Did you verify that each of these actually work properly? I know that 29-30 did (I tested when I added them). But for 31-34, I have no idea if these work or not.

We need to consider with these:

1. Do we want to leave them in here (suggesting that users can adjust them)?
2. If yes, then we must document what these do, exactly, and we must make sure each of them actually work!

We definitely should include NoSSLv2 and NoSSLv3, although they too should be commented out (verify that the current values are actually default values), and we should include a brief description of what they mean. It think it's obvious based on the name, but then I worked on it, I'm not at all sure that customers would understand that.

Reply:
will update them later today, thanks.